plugins/lsp/conform-nvim.nix

```diff
@@ -36,97 +110,112 @@ in
             };
         ```
```

Comment:
Sorry, I missed this in the initial reviews. This description is basically entirely example code with no actual text... I don't _mind_ embedding "contextual" examples like this in a description, but I prefer to also have some _actual_ description along with an _actual_ `example` value.

Don't need to make it perfect though, bare minimum I think we should have a short option description followed by a short explanation of the example usage.

Reply:
Ah, yeah. Forgot to clean that up after I updated the example inside. I'll add a comment to the top describing the option.